### ISSUE TITLE
Need to output non intrusive polynomial chaos coefficients

### ISSUE THREAD

**Issue by MaxAlgerMeyer** [jonathf/chaospy#355]
**Describe your problem**
We are using chaospy to construct the non intrusive polynomial chaos surrogate models and are looking for the mean, standard deviation, and polynomial chaos expansion coefficients. We noticed that the output from chaospy gives us the polynomial equations, but none of the polynomial equation coefficients equal the mean value, which suggests that these coefficients are the product of the random basis function (psi) and the deterministic component (alpha).  We were wondering if there is a way to output just the PCE coefficients? 
**Initial implementation**
We've looked through the code but can't seem to find where the PCE coefficients are computed.

**Additional context**
Reference file attached, in particular we are working with the material in section III, and specifically we are referring to equation 5. 
[Recent Advances in Non-Intrusive Polynomial Chaos and Stochastic Collocation Methods for Uncertainty Analysis and Design.pdf](https://github.com/jonathf/chaospy/files/7288928/Recent.Advances.in.Non-Intrusive.Polynomial.Chaos.and.Stochastic.Collocation.Methods.for.Uncertainty.Analysis.and.Design.pdf)

**Comment by jonathf** [comment]
Yes, depending on the method you are using. For Stochstic collocation you can retrieve the coefficients from:
```python
approx_model, coefficients = chaospy.fit_regression(..., retall=True)
```
In the docs I refer to them as Fourier coefficients, which i likely why you didn't find them. :)
